src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts

```diff
@@ -84,4 +84,23 @@ export class InputActionDialog {
   close() {
     this.modal.close();
   }
+
+  checkIfJenkinsIsUp() {
+    let url = 'https://jenkins.openshift.io';
+    this.http.get(url)
+      .subscribe((response) => {
+        if (response.status == 200) {
+          console.log('Got ' + response.status + ', Jenkins is up and running...');
+          this.invokeUrl(this.inputAction.proceedUrl);
+          return;
+        } else if (response.status == 307 || response.status == 302) {
+          console.log('Got ' + response.status + ', Jenkins Proxy is logging you in...');
```

Comment:
No indication in the UI that this happened?

Reply:
@gastaldi We could probably use NotificationService to show these messages to user rather than just logging it to console. @kishansagathiya What do you think?

Reply:
yes some kind of feedback will be great so user knows that he has pushed the button and we are working on it.

Reply:
lets not reveal internal implementation details like "Jenkins proxy" to user in the UI here. 

Better to just say "Connecting to Jenkins..." or similar.